src/dependency/Pyegi/Pyegi/installer.py

```diff
@@ -0,0 +1,169 @@
+import os
+from os.path import exists
+import sys
+import toml
+from datetime import datetime as dt1
+import shutil
+import csv
+import json
+
+start_time = dt1.now()
+print("\nStart time: " + str(start_time) + "\n")
+
+temp_dir = os.path.dirname(__file__) + "/temp/"
+commons_dir = os.path.dirname(__file__) + "/commons/"
+dependency_dir = os.path.dirname(os.path.dirname(__file__)) + "/"
+scriptsPath = dependency_dir + "PythonScripts/"
+# system_inputs = sys.argv
+
+scripts_names = [
+    name
+    for name in os.listdir(scriptsPath)
+    if os.path.isdir(os.path.join(scriptsPath, name))
+]
+
+
+def create_dirs(path):
+    parent, _ = os.path.split(path)
+    if not exists(parent):
+        os.makedirs(parent)
+
+
+def install_pkg(script):
+    print(f"Processing {script} dependencies...")
+    script_path = scriptsPath + script + "/"
+    if exists(script_path + "pyproject.toml"):
+        os.chdir(script_path)
+        os.system("poetry lock")
+        src = script_path + "pyproject.toml"
+        dst = temp_dir + "pyproject.toml"
+        shutil.copyfile(src, dst)
+        os.chdir(temp_dir)
+        os.system("poetry lock")
+        lock_content = toml.load("poetry.lock")
+        packages = lock_content["package"]
+        packages_to_move = []
+        phase1_start_time = dt1.now()
+        for package in packages:
+            name_in_commons = f"{package['name']}-{package['version']}"
+            f = open(commons_dir + "lib_links.json")
+            lib_links = json.load(f)
+            f.close()
+            new_package = True
+            for pkg in lib_links["Packages"]:
```

Comment:
it's better to be "packages" for consistency; other names in the file all starts with a lowercase

Reply:
That's exactly because of consistency within all the json files which we're using. Within all the other json files the fields are capitalized.

Reply:
`scripts`, `name`, and other fields in this specific json file are all uncapitalized.
![image](https://user-images.githubusercontent.com/25867589/181256748-59d993d7-57cb-4c5b-996b-3712d8c31f21.png)

Reply:
oh sh*t. u'r right. I forgot to capitalize those two